### DISCUSSION TITLE
Cursor lags when I press the pen in osu!(lazer)

### DISCUSSION THREAD

**Discussion by ilyuh** [ppy/osu#22560] [Q&A]
### Type

Performance

### Bug description

When I have "Raw Input" on and whenever I am pressing the pen my cursor starts lagging.
When I turn "Raw Input" off the problem disappears but it makes the input less precise.
There is no such problem with the classic osu! client. Also there was no problem with older versions of osu!(lazer) but unfortunately I have not played the game for a while so I cannot say when the problem first appeared.
I am using a Gaomon S620 tablet with Gaomon_Windows_Driver_v14.8.133.1259 installed ("Game Mode" on, "Use Windows Ink" off). I am also using the latest versions of the game clients available.

### Screenshots or videos

https://user-images.githubusercontent.com/43908503/217071887-163306e4-d7b9-4842-9827-e887856972ad.mp4

### Version

2023.123.0-lazer

### Logs

[performance.log](https://github.com/ppy/osu/files/10668728/performance.log)
[runtime.log](https://github.com/ppy/osu/files/10668729/runtime.log)
[updater.log](https://github.com/ppy/osu/files/10668730/updater.log)
[database.log](https://github.com/ppy/osu/files/10668731/database.log)
[input.log](https://github.com/ppy/osu/files/10668732/input.log)
[network.log](https://github.com/ppy/osu/files/10668733/network.log)

**Comment by Bloomdw** [comment]
I'm experiencing exactly the same issue. I never had to turn off raw input in osu!stable and it really makes playing so much harder.

**Comment by Stoppedpuma** [comment]
Raw input shouldn't make any difference for tablets as all it does it bypass acceleration, unless you have acceleration somewhere in your tablet configuration or enhanced cursor precision (which I don't believe even works with tablets) then you're experiencing placebo.
